app/assets/stylesheets/application.scss

```diff
@@ -17,5 +17,48 @@
 @import "bootstrap";
 
 body {
-  background-color: white;
+  background-color: #e5e8e0;
+}
+
+.container-fluid { max-width: 80%;
+
+}
+
+img {width: 100%;
+
+}
+
+.featured {
+  background-color: #c3a17d;
```

Comment:
Color literals like `#c3a17d` should only be used in variable declarations; they should be referred to via variable everywhere else.